licenses/management/commands/publish.py

```diff
@@ -0,0 +1,148 @@
+    cmd = "python manage.py distill-local --quiet --force"
+    return subprocess.run(
+        cmd, shell=True, check=True, text=True, input="YES", env=my_env,
+    )
+
+
+def git_branch_status(branch: str):
+    """Checks if there is a build to commit using git status
+
+    Returns True or False
+    """
+    check_branch_status_cmd = (
+        GO_TO_TRANSLATIONS_REPO + f"git checkout {branch} && " + "git status"
+    )
+    status = (
+        subprocess.check_output(check_branch_status_cmd, shell=True)
+        .decode()
+        .split("\n")
+    )
+    if "nothing to commit, working tree clean" in status:
+        return False
+    return True
+
+
+def git_on_branch_and_pull(branch: str):
+    """Returns a string represention of the git command to checkout and pull from a branch"""
+    git_on_branch_and_pull_cmd = GO_TO_TRANSLATIONS_REPO + (
+        f"git checkout {branch} && git pull origin {branch}"
+    )
+    return subprocess.run(git_on_branch_and_pull_cmd, shell=True, check=True)
+
+
+def git_commit_and_push(branch: str):
+    """Returns a string representation of the git command to checkout, commit, and push branch"""
+    commit_and_push_cmd = GO_TO_TRANSLATIONS_REPO + (
+        "git add build/ && "
+        f"git checkout {branch} && "
+        f"git commit -m '{branch}' && "
+        f"git push origin {branch}"
+    )
+    return subprocess.run(commit_and_push_cmd, shell=True, check=True,)
+
+
+def pull_translations_branches():
+    """Git pulls branches in cc-licenses-data to update local git registry"""
+    return subprocess.run(PULL_DOWN_TRANSLATION_BRANCHES, shell=True, check=True,)
+
+
+def list_open_branches():
+    """List of open branches in cc-licenses-data repo
+    """
+    list_branches_cmd = GO_TO_TRANSLATIONS_REPO + "git branch --list"
+    pull_translations_branches()
+    branches = (
+        subprocess.check_output(list_branches_cmd, shell=True).decode().split("\n")
+    )
+    print("\n\nWhich branch are we publishing to?\n")
+    [print(b) for b in branches]
+
+
+def publish_branch(branch: str):
+    """Workflow for publishing a single branch"""
+    git_on_branch_and_pull(branch)
+    run_django_distill()
+    build_to_push = git_branch_status(branch)
+    if build_to_push:
+        git_commit_and_push(branch)
+    else:
+        print(f"\n{branch} build dir is up to date.\n")
+
+
+def publish_all():
+    """Workflow for checking branches other than develop and updating their build dir
+
+    Develop is not checked because it serves as the source of truth. It should be
+    manually merged into.
+    """
+    list_branches_cmd = GO_TO_TRANSLATIONS_REPO + "git branch --list"
+    branches = (
+        subprocess.check_output(list_branches_cmd, shell=True).decode().split("\n")
+    )
+    exclude_develop_list = [b for b in branches if "develop" not in b]
+    del exclude_develop_list[-1]
+    branch_list = [b.lstrip() for b in exclude_develop_list]
```

Comment:
If the branch list output has whitespace, why don't we strip it off as soon as possible, then we'll just have a list of the exact branch names.  We might even write a little utility method that just returns the cleaned up list of all the branch names, and then here we can do what we want with it without worrying here about those details.

Reply:
added utility functions